packages/components/src/Input/Input.stories.tsx

```diff
@@ -26,6 +26,8 @@ const meta = {
         'time',
         'url',
         'week',
+        'file',
```

Comment:
Can you also make a separate story to show them! :)

Reply:
@sebald 
already did that in this commit but i deleted them  :D 
https://github.com/marigold-ui/marigold/pull/3244/commits/e1dc6ab8c0fa08e27e01496695c16f1fdf14527a
i will get them back :100: